Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.9.1-buster
+
+WORKDIR /opt/app/
+EXPOSE 5001
+ADD requirements.txt .
+RUN pip install -r requirements.txt
+RUN mkdir /root/polyglot_data
+RUN polyglot download ner2.en ner2.de  ner2.ar ner2.bg ner2.ca ner2.cs ner2.da ner2.el ner2.es ner2.et ner2.fa ner2.fi ner2.fr ner2.he ner2.hi ner2.hr ner2.hu ner2.id ner2.it ner2.ja ner2.ko ner2.lt ner2.lv ner2.ms ner2.nl ner2.no ner2.pl ner2.pt ner2.ro ner2.ru ner2.sk ner2.sl ner2.sr ner2.sv ner2.th ner2.tl ner2.tr ner2.uk ner2.vi ner2.zh
```

Comment:
Put this and the next line in a bash / python script / command that uses some FOR loop - I don't want to see us touching this by hand.